File: Freight.API/Libraries/Freight.API.Common/Model/Item/Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Freight.API.Common.Model.Item
{
    public class Item
    {
        public int ItemID { get; set; }
        public int BusinessID { get; set; }
        public int ShipperID { get; set; }
        public string ItemNumber { get; set; }
        public string ItemName { get; set; }
        public string HTSCode { get; set; }
        public string FNSKU { get; set; }
        public string ASIN { get; set; }
        public decimal? Weight { get; set; }
        public string WeightUnit { get; set; }
        public decimal? UnitPrice { get; set; }
        public string Currency { get; set; }
        public DateTime? UpdatedOn { get; set; }
        public DateTime CreatedOn { get; set; }
    }
}

File: Freight.Web/freight/src/app/_shared/model/item.ts
export class Item {
  constructor(
    public itemID: number,
    public businessID: number,
    public shipperID: number,
    public itemNumber: string,
    public itemName: string,
    public htsCode: string,
    public fnsku: string,
    public asin: string,
    public weight: number,
    public weightUnit: string,
    public unitPrice: number,
    public currency: string,
    public updatedOn: Date,
    public createdOn: Date,
  ) { }
}